.gitignore

```diff
@@ -1,12 +1,30 @@
-*.sql
-/.vscode
-/venv
-*.swp
+# Cache
 __pycache__
 /nyaa/static/.webassets-cache
+
+# Virtual Environments
+/venv
+
+# Coverage
+.coverage
+/htmlcov
+
+# Editos / IDEs
+/.vscode
+/.idea
+
```

Comment:
IDEs' `gitignore`s are generally meant to (and preferably) be set on git's global configuration.